### DISCUSSION TITLE
Why can't I access my own project?

### DISCUSSION THREAD

**Discussion by unovil** [supabase/supabase#14781] [Questions]
As said in the title, I can't access my project on the dashboard. I can't change any settings because I'm told I would need additional permissions, even though I am the owner of the project. Additionally, everything (including settings of the project dashboard, and even the organization information and settings on the dashboard) is loading forever. (Note that the project has been paused, although the project's been dormant multiple times before and I've always restored the project without encountering any errors.)

Any help would be greatly appreciated T_T
![image](https://github.com/supabase/supabase/assets/70586029/ed25ba3f-412f-4eb0-b6d4-17683742901d)
![image](https://github.com/supabase/supabase/assets/70586029/7d1e4692-9c80-4a08-845d-4497dce58799)

**Comment by GaryAustin1** [comment]
You probably should contact support.

**Reply by unovil** [reply]
I'll try, thanks!
